Cargo.toml

```diff
@@ -39,6 +50,6 @@ default-features = false
 optional = true
 
 [dependencies.multipart]
```

Comment:
the good part about depending on reqwest is that we don't need this or the tls crates at all, both implicitly do the right thing